src/plugins/pii-scrub/pii-scrub.js

```diff
@@ -213,6 +213,11 @@ var $document = wb.doc,
 
 		// Add PII fields HTML if using a custom UI template
 		if ( modalTemplate ) {
+
+			// Temporary fix for implementers that added the "popup-modal-dismiss" class to the submit button
+			// Deprecate on version 18.0.0 of GCWeb and Remove on version 19.0.0
```

Comment:
Not sure if we will be able to get this line removed eventually, we are in wet-boew not GCWeb. There is no plan to apply any breaking change to wet-boew.

I think this comment need to be reviewed consider that.

I suggest to keep that workaround to avoid a breaking change but don't mention any removal version.